miden-lib/asm/miden/kernels/tx/memory.masm

```diff
@@ -41,7 +41,7 @@ const.ACCT_ID_PTR=101
 const.INIT_ACCT_HASH_PTR=102
 
 # The memory address at which the global nullifier commitment is stored
-const.NULLIFIER_COM_PTR=103
```

Comment:
I expanded the constants names, here COM -> COMMITMENT, it does make the text longer, but in some cases (like the prepare_transaction) it makes it easier to understand what is going on (and removing conflicts, with things like PH for proof hash and previous hash)